moveit_ros/visualization/rviz_plugin_render_tools/CMakeLists.txt

```diff
@@ -22,17 +22,19 @@ add_library(${MOVEIT_LIB_NAME} SHARED
   src/trajectory_visualization.cpp
   src/trajectory_panel.cpp
   src/mesh_shape.cpp
+  ${HEADERS}
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
 target_link_libraries(${MOVEIT_LIB_NAME}
     rviz_ogre_vendor::OgreMain
     Qt5::Widgets
+    rviz_common::rviz_common
```

Comment:
Why not use `ament_target_dependencies()` for these dependencies?

Reply:
The rviz_default_plugins package does the linking very explicitly as well. Do we know why? It might just be because rviz2 was ported pre-crystal.

Reply:
I don't know why they did it this way, but using `rviz_default_plugins` gives linking error, I looked at how it's defined in the generated `.cmake` files and the target name is `rviz_default_plugins::rviz_default_plugins`

Reply:
rviz_default_plugins contains a number Displays. I wasn't saying we should link against that library, just that we could use it at a template for creating custom Displays.